### PR TITLE
Add and feature Seminar page

### PR DESCRIPTION
**Pull request by lennartdefacto**
- [x] Create Seminar page
- [x] Add Seminar to Event page
- [x] Feature Seminar on homepage
- [x] Add social image
- [ ] Add seminar information as content

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
- [ ] Use unique formkeep endpoint or fallback
- [ ] Add header images

File: source/localizable/events/_events.nl.md.erb [hunk 1]
```diff
@@ -20,4 +20,25 @@
       <%= link_to "Meer informatie & aanmelden", "/events/gebruikersbijeenkomst-november-2018.html", class: "button pink" %>
     </div>
   </li>
+  <li class="event">
+    <% link_to "/events/seminar-november-2018.html", class: "col left" do %>
+      <h2>
+        Defacto <br> Seminar
```
Comment (fatboypunk) [inline source/localizable/events/_events.nl.md.erb]: wut? is this normal?

Comment (fatboypunk) [reply source/localizable/events/_events.nl.md.erb]: can we atleast have Seminar on a new line?

File: source/localizable/events/seminar-november-2018.html.erb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+          Inloop met lunch
+        </li>
+        <li>
+          Opening door Hans Veeger / spreker X
+        </li>
+        <li>
+          De visie achter CAPP LMS, door Hugo Veeger
+        </li>
+        <li>
+          Het gebruik van CAPP LMS door een aantal van onze klanten: </br>
```
Comment (fatboypunk) [inline source/localizable/events/seminar-november-2018.html.erb]: `<br />` could work but `<br>` should be fine :)

File: source/localizable/events/_events.nl.md.erb (hunk 1 above)
Comment (RonaldDijkstra) [reply source/localizable/events/_events.nl.md.erb]: Can be 1 line

File: source/localizable/events/seminar-november-2018.html.erb (hunk 2 above)
Comment (RonaldDijkstra) [reply source/localizable/events/seminar-november-2018.html.erb]: screen shot 2018-09-13 at 14 49 02

A list in a list is what's best here I think.

File: source/localizable/events/seminar-november-2018.html.erb [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+<section class="intro">
+  <div class="row">
+    <article>
+      <p>
+        We nodigen je van harte uit om bij het seminar van Defacto aanwezig te zijn. Tijdens dit seminar zullen we vanuit het thema <strong>'Het nieuwe LMS: via nieuwe wegen van verplicht vakbevoegd naar gemotiveerd vakbekwaam'</strong> laten zien wat ons <a href="/capp-lms/">CAPP LMS</a> voor organisaties kan betekenen. Een LMS dat volledig inhouse door Defacto is ontwikkeld en gebouwd op jaren aan ervaring en onze leidende rol als het gaat om het gebruik van de nieuwste technieken en methodieken. We zijn ervan overtuigd en willen dan ook graag laten zien dat CAPP LMS organisaties zal helpen bij het behalen van doelstellingen op het gebied van het laten leren en ontwikkelen van medewerkers!
+      </p>
+      <ul>
+        <li>
+          Het programma duurt van 12:30 tot 17:00 met afsluitend een borrel
```
Comment (sn3p) [inline source/localizable/events/seminar-november-2018.html.erb]: Line 21 states `18:00`, or is this including the borrel?

File: source/localizable/events/seminar-november-2018.html.erb [hunk 4]
```diff
@@ -18,7 +18,7 @@ lang_ignore: en, de
         Via nieuwe wegen van verplicht vakbevoegd naar gemotiveerd vakbekwaam
       </h1>
       <p>
-        22 november 2018 | 12.30 - 18.00<br />
+        22 november 2018 | 12.30 - 17.00<br>
```
Comment (sn3p) [inline source/localizable/events/seminar-november-2018.html.erb]: Do we want a colon `:` here instead of a period `.`? (same goes for the other events)

Comment (RonaldDijkstra) [reply source/localizable/events/seminar-november-2018.html.erb]: We want colon @lennartdefacto

File: source/localizable/evenementenagenda/workshop-innovative-lernumgebung.html.erb [hunk 5]
```diff
@@ -75,7 +77,7 @@ lang_ignore: en, nl
         Campus Großhadern<br />
         Marchioninistraße 15 <br />
         81377 München<br />
-        <a href="https://www.google.nl/maps/place/Klinikum+Gro%C3%9Fhadern/@48.1113273,11.4671084,17z/data=!3m1!4b1!4m5!3m4!1s0x479dd84e174f70c5:0xbe2856eb1d70bdbb!8m2!3d48.1113273!4d11.4692971" target="blank" title="Klinikum Großhadern - Universität München">Google Maps</a>
+        <a href="https://www.google.nl/maps/place/Klinikum+Gro%C3%9Fhadern/@48.1113273,11.4671084,17z/data=!3m1!4b1!4m5!3m4!1s0x479dd84e174f70c5:0xbe2856eb1d70bdbb!8m2!3d48.1113273!4d11.4692971" target="_blank" title="Klinikum Großhadern - Universität München">Google Maps</a>
```
Comment (RonaldDijkstra) [inline source/localizable/evenementenagenda/workshop-innovative-lernumgebung.html.erb]: I did this but should've been in a separate PR.

**Pull request closed without merging**